From ee6fb0d9de661f6893e204586d16554ed6c38d5f Mon Sep 17 00:00:00 2001
From: Christophe Kerello <christophe.kerello@foss.st.com>
Date: Fri, 8 Dec 2023 09:20:52 +0100
Subject: [PATCH 0496/1695] dts: stm32: fix PI8 RIF configuration on STM32MP25
 DK board

The fix solves following issue on Kernel side.

stm32mp257-pinctrl soc@0:pinctrl@44240000: Can't access gpio 136
stm32mp257-pinctrl soc@0:pinctrl@44240000: request() failed for pin 136
stm32mp257-pinctrl soc@0:pinctrl@44240000: pin-136 (46000000.rtc) status -13
stm32mp257-pinctrl soc@0:pinctrl@44240000: could not request pin 136 (PI8) from group PI8  on device soc@0:pinctrl@44240000

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git aa976d10d21b5e2b28d7e18bbdf92cedafbf73eb dts: stm32: fix PI8 RIF configuration on STM32MP25 DK board]
Change-Id: If14a9b544114505ef5f23e65f8600144c56be5a5
Signed-off-by: Christophe Kerello <christophe.kerello@foss.st.com>
---
 core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rif.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rif.dtsi b/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rif.dtsi
index 4677f82f6..bafd602e5 100644
--- a/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rif.dtsi
+++ b/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rif.dtsi
@@ -500,7 +500,7 @@
 		RIFPROT(RIF_IOPORT_PIN(5), RIF_UNUSED, RIF_UNLOCK, RIF_NSEC, RIF_NPRIV, RIF_UNUSED, RIF_SEM_DIS, RIF_CFDIS)
 		RIFPROT(RIF_IOPORT_PIN(6), RIF_UNUSED, RIF_UNLOCK, RIF_NSEC, RIF_NPRIV, RIF_UNUSED, RIF_SEM_DIS, RIF_CFDIS)
 		RIFPROT(RIF_IOPORT_PIN(7), RIF_UNUSED, RIF_UNLOCK, RIF_NSEC, RIF_NPRIV, RIF_UNUSED, RIF_SEM_DIS, RIF_CFDIS)
-		RIFPROT(RIF_IOPORT_PIN(8), RIF_UNUSED, RIF_UNLOCK, RIF_SEC, RIF_PRIV, RIF_CID1, RIF_SEM_DIS, RIF_CFEN)
+		RIFPROT(RIF_IOPORT_PIN(8), RIF_UNUSED, RIF_UNLOCK, RIF_NSEC, RIF_NPRIV, RIF_UNUSED, RIF_SEM_DIS, RIF_CFDIS)
 		RIFPROT(RIF_IOPORT_PIN(9), RIF_UNUSED, RIF_UNLOCK, RIF_NSEC, RIF_NPRIV, RIF_UNUSED, RIF_SEM_DIS, RIF_CFDIS)
 		RIFPROT(RIF_IOPORT_PIN(10), RIF_UNUSED, RIF_UNLOCK, RIF_NSEC, RIF_NPRIV, RIF_UNUSED, RIF_SEM_DIS, RIF_CFDIS)
 		RIFPROT(RIF_IOPORT_PIN(11), RIF_UNUSED, RIF_UNLOCK, RIF_NSEC, RIF_NPRIV, RIF_UNUSED, RIF_SEM_DIS, RIF_CFDIS)
-- 
2.47.2

